### DISCUSSION TITLE
Do you guys mix and match CT's??

### DISCUSSION THREAD

**Discussion by jbuszkie** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#172] [Q&A]
I haven't touched my energy meter in some time.  But now I want to add more monitoring.  I have to re-learn what I have to get.  I want to measure the current of my heat pump.  I want a 30-50A CT with , I guess, no burden resistor.  I only see like 50A/1V ones..  The only mA ones  (from YHDC) are the 100A/50mA.  Do most of you guys have a mix and match of CT's?  How do you remember which ones have the burden resistor cut on the board?  I know I will forget and possibly plug a 50mA version into a slot where I cut the burden resistor!!!

**Comment by jbuszkie** [comment]
After looking at what I have already...  For, at least, one CT, I must have removed the burden resistors on the CT clamp because I scratched out 1V and wrote 50mA...  So may be that's what I did.  After a little searching, it's easy enough to remove those resistors. So maybe that's the route I'll go.

**Comment by kaijk** [comment]
I maintain a spreadsheet documenting for each main or add on board's CT ID,  the CT Circuit Name and Rating, and whether the Burden Resistor has been severed. 

Each CT then has a "Dymo" label with its monitored circuit name and the board's CT ID noting the main or add on-board on which it resides.  

The source `yaml` file also has comments about this as well.

**Comment by TRSx80** [comment]
I keep extensive Orgmode text file notes about pretty much everything, but especially about projects.  And most especially, those with critical information, that I don't look at frequently...
